src/ansys/dpf/core/operators/math/expansion_psd.py

```diff
@@ -0,0 +1,399 @@
+"""
+expansion_psd
+
+Autogenerated DPF operator classes.
+"""
+
+from warnings import warn
+from ansys.dpf.core.dpf_operator import Operator
+from ansys.dpf.core.inputs import Input, _Inputs
+from ansys.dpf.core.outputs import Output, _Outputs
+from ansys.dpf.core.operators.specification import PinSpecification, Specification
+
+
+class expansion_psd(Operator):
+    """Computes the PSD response for one-sigma solution.
```

Comment:
Hi @oparreno could we maybe have a more extensive description of the goal of the operator here?